### PR TITLE
Set alpine:latest as a base image by default

### PR DESCRIPTION
**Pull request by e0ne**
Image size changed from 346MB to 50.3MB

Signed-off-by: Ivan Kolodiazhnyi <ikolodiazhny@nvidia.com>

### CODE REVIEW

**Comment by moshe010** [comment]
@e0ne  do you understand why CI failed on this change?

**Comment by e0ne** [comment]
@moshe010 failures aren't related to the change, need to dig deeper

File: README.md [hunk 1]
```diff
@@ -305,6 +306,16 @@ Example can be found under `./example/crs/mellanox.com_v1alpha1_macvlannetwork_c
 
 A deployment example can be found under `example` folder [here](https://github.com/Mellanox/network-operator/blob/master/example/README.md).
 
+## Docker image
+Network operator uses `alpine` base image by default. To build Network operator with
+another base image you need to pass `BASE_IMAGE` argument:
+
+```
+docker build -t network-operator \
+--build-arg BASE_IMAGE=registry.access.redhat.com/ubi8/ubi-minimal:latest \
```
Comment (moshe010) [inline README.md:315]: you need to add to Travis to build both type of images (not for this release)

Comment (e0ne) [reply README.md:320]: We'll move it to jenkins to not hit dokerpull limit. golang image from dokcerhub is still using for base builder image

**Comment by moshe010** [comment]
@e0ne  where do you push both images to dockerhub?

**Review by moshe010 — APPROVED** [review]

**Pull request merged**